src/devices/Bmxx80/Bme680HeaterProfileConfig.cs

```diff
@@ -33,7 +34,7 @@ public class Bme680HeaterProfileConfig
         /// <param name="heaterResistance">The heater resistance in Ohm.</param>
         /// <param name="heaterDuration">The heating duration in ms.</param>
```

Comment:
description

Reply:
I've likely missed some cases, please re-view all descriptions in this PR to double check they don't contain explicit units